op-chain-ops/cmd/celo-migrate/state.go

```diff
@@ -219,3 +236,54 @@ func applyAllocsToState(db *state.StateDB, genesis *core.Genesis) error {
 	log.Info("Migrated OP contracts into state DB", "copiedAccounts", accountCounter, "overwrittenAccounts", overwriteCounter)
 	return nil
 }
+
+// setupDistributionSchedule sets up the distribution schedule contract with the correct balance
+// The balance is set to the difference between the ceiling and the total supply of the token
+func setupDistributionSchedule(db *state.StateDB, config *params.ChainConfig) error {
+	log.Info("Setting up CeloDistributionSchedule balance")
+
+	if !db.Exist(celoDistributionScheduleAddress) {
+		log.Error("DistributionSchedule account does not exist", "stash", celoDistributionScheduleAddress)
```

Comment:
If we want to get rid of the "stash" terminology (not sure if we do), we should also remove it here.